commands/object-encoding.md

```diff
@@ -2,7 +2,12 @@ Returns the internal encoding for the Redis object stored at `<key>`
 
 Redis objects can be encoded in different ways:
 
-* Strings can be encoded as `raw` (normal string encoding) or `int` (strings representing integers in a 64 bit signed interval are encoded in this way in order to save space).
+* Strings can be encoded as: 
+
+	- `raw`, normal string encoding.
+	- `int`, strings representing integers in a 64-bit signed interval, encoded in this way to save space.
+	- `embstr`, an embedded string, which is an object where the internal simple dynamic string, `sds`, is an unmodifiable string allocated in the same chuck as the object itself. `embstr` can be strings with lengths up to the hardcoded limit of `OBJ_ENCODING_EMBSTR_SIZE_LIMIT` or `44` bytes. 
```

Comment:
* I repeat the style guideline "one sentense per line". :-)
* 44 doesn't need `44` formatting. It's just a number, not code.
* Are these tabs? I'm not sure we want tabs for indentation. I changed to 4 spaces below.

```suggestion
    - `raw`, normal string encoding.
    - `int`, strings representing integers in a 64-bit signed interval, encoded in this way to save space.
    - `embstr`, an embedded string, which is an object where the internal simple dynamic string, `sds`, is an unmodifiable string allocated in the same chuck as the object itself.
      `embstr` can be strings with lengths up to the hardcoded limit of `OBJ_ENCODING_EMBSTR_SIZE_LIMIT` or 44 bytes. 
```